scipy/special/add_newdocs.py

```diff
@@ -6811,12 +6811,42 @@ def add_newdoc(place, name, doc):
 
     .. math:: \\zeta(x) = \\sum_{k=2}^{\\infty} 1 / k^x,
 
-    where ``x > 1``.
+    where ``x > 1``.  For ``x < 1``, the analytic continuation is computed.
+
+    Because of limitations of the numerical algorithm, ``zetac(x)`` returns
+    `nan` for `x` less than -30.8148.
+
+    Parameters
+    ----------
+    x : array_like of float
+        Values at which to compute zeta(x) - 1 (must be real).
+
+    Returns
+    -------
+    float
```

Comment:
Array-like of float (to match input description, since it should be the same shape right?)

Reply:
Sure.  There doesn't appear to be a consistent style for describing the return value of ufunc-like functions: `ndarray`, `array`, `float`, `array_like` are some of the variations.  (Personally I've never liked `array_like`.  Why use an underscore?  There is no such Python object, it is not an identifier, so why not use the normal dash, and call it "array-like"?  But I'll begrudgingly use it--choose your battles and all that.)